src/__tests__/useThrottle.test.ts

```diff
@@ -0,0 +1,16 @@
+import { renderHook, act } from '@testing-library/react-hooks';
+import useThrottle from '../useThrottle';
+
+it('should throttle the given value by specific time', () => {
+  jest.useFakeTimers();
+  const { result, rerender, unmount } = renderHook(props => useThrottle(props, 1000), { initialProps: 0 });
+  expect(result.current).toBe(0);
+  rerender(1);
+  expect(result.current).toBe(0);
+  act(() => jest.advanceTimersByTime(1000));
+  expect(result.current).toBe(1);
+  act(() => jest.advanceTimersByTime(1000));
+  rerender(2);
+  act(() => jest.advanceTimersByTime(500));
+  unmount();
```

Comment:
Can you split this single test into different ones? So it's more clear what you are testing on it, like "should clear pending timeouts on unmount". Also it would be interesting to test additional cases like rerendering the component with different ms prop or check the result is same as before if time is advanced less than expected ms.